### PR TITLE
fix: add toc

### PR DESCRIPTION
**Pull request by FionnD**
re-adding back

### CODE REVIEW

**Comment by FionnD** [comment]
For some reason, I can not get the TOC to work correctly with Emojis. It only works with plain text. 

Any suggestion @alexcg1 @theUnkownName ?


Screenshot 2021-05-17 at 10 45 24

File: wikipedia-sentences/README.md [hunk 1]
```diff
@@ -6,13 +6,15 @@
 
 ## Table of contents: 
 
+- [Overview](#overview)
+- [🐍 Build the app with Python](#-🐍-build-the-app-with-python)
```
Comment (theUnkownName) [inline wikipedia-sentences/README.md:10]: Remove the emoji from the `(#build-the-app-with-python)` I think that's the problem

Comment (FionnD) [reply wikipedia-sentences/README.md:10]: That's the situation [here](https://github.com/jina-ai/examples/tree/master/multires-lyrics-search) and it's not working.

Comment (alexcg1) [reply wikipedia-sentences/README.md:10]: Are you manually creating toc? If so you can hover over the heading of the section and it should show a link icon on the left. Then you can right-click and copy link

Comment (FionnD) [reply wikipedia-sentences/README.md]: that shows the full url? 

`https://github.com/jina-ai/examples/blob/9d9de0bade9bb6498c266aab3f6bfdb4cba3ddf2/wikipedia-sentences/README.md#-build-the-app-with-python`

Comment (FionnD) [reply wikipedia-sentences/README.md]: I tried just adding the ending #-build-the-app-with-python but it didn't work

File: wikipedia-sentences/README.md [hunk 2]
```diff
@@ -7,15 +7,15 @@
...
-
+- [🐍 Build the app with Python](#-build-the-app-with-python)
+- [🔮 Overview of the files in this example](#-overview-of-the-files-in-this-example)
+- [🌀 Flow diagram](#-flow-diagram)
+- [🔨 Next steps, building your own app](#-next-steps-building-your-own-app)
+- [🐳 Deploy the prebuild application using Docker](#-deploy-the-prebuild-application-using-docker)
+- [🙍 Community](#-community)
+- [🦄 License](#-license)
+
+#-step-1-clone-the-repo-and-install-jina
```
Comment (alexcg1) [inline wikipedia-sentences/README.md:18]: Delete this

**Review by alexcg1 — CHANGES_REQUESTED** [review]

**Review by alexcg1 — DISMISSED** [review]
LGTM :+1:

**Review by alexcg1 — APPROVED** [review]

**Pull request merged**